docs/rackhd/event_notification.rst

```diff
@@ -420,6 +420,68 @@ If the endpoint is redfish enabled and supports the Resfish EventService, it is
 		"Protocol": "Redfish"
 	}
 
+If the node is a Dell node, it is possible to post the Graph.Dell.Configure.Redfish.Alerting workflow.
+The workflow will:
+
+1- Enable Alerts for the Dell node. Equivalent to running "set iDRAC.IPMILan.AlertEnable 1" racadam command.
+
+2- Enable redfish alerts. Equivalent to running "eventfilters set -c idrac.alert.all -a none -n redfish-events" racadam command.
+
+3- Disable the "Audit" info alerts. Equivalent to running  "eventfilters set -c idrac.alert.audit.info -a none -n none" racadam command.
+
+The workflow will run the default values if the node's obm is set and the "rackhdPublicId" property is set in the rackHD config.json file.
```

Comment:
rackhdPublicId  should be 'rackhdPublicIp' according to https://github.com/RackHD/RackHD/pull/741